src/content/events/2024/bytesize_Tamper_Monkey.md

```diff
@@ -1,24 +0,0 @@
----
-title: 'Bytesize: Tamper Monkey'
-subtitle: Phil Ewels, Seqera
-type: talk
-startDate: '2024-05-14'
```

Comment:
Just change the date rather than deleting?

Reply:
I kept a copy of it on my computer, was just waiting for you to give me a new date.